shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java

```diff
@@ -581,6 +503,149 @@ public SystemChromeStyle(
+              // NoSuchFieldException: One or more expected fields were either omitted or referenced an invalid type.
+              result.error("error", exception.getMessage(), null);
+            }
+            break;
+          case "SystemChrome.setApplicationSwitcherDescription":
+            try {
+              AppSwitcherDescription description = decodeAppSwitcherDescription((JSONObject) arguments);
+              platformMessageHandler.setApplicationSwitcherDescription(description);
+              result.success(null);
+            } catch (JSONException exception) {
+              // One or more expected fields were either omitted or referenced an invalid type.
+              result.error("error", exception.getMessage(), null);
+            }
+            break;
+          case "SystemChrome.setEnabledSystemUIOverlays":
+            try {
+              List<SystemUiOverlay> overlays = decodeSystemUiOverlays((JSONArray) arguments);
+              platformMessageHandler.showSystemOverlays(overlays);
+              result.success(null);
+            } catch (JSONException | NoSuchFieldException exception) {
+              // JSONException: One or more expected fields were either omitted or referenced an invalid type.
+              // NoSuchFieldException: One or more of the overlay names are invalid.
+              result.error("error", exception.getMessage(), null);
+            }
+            break;
+          case "SystemChrome.restoreSystemUIOverlays":
+            platformMessageHandler.restoreSystemUiOverlays();
+            result.success(null);
+            break;
+          case "SystemChrome.setSystemUIOverlayStyle":
+            try {
+              SystemChromeStyle systemChromeStyle = decodeSystemChromeStyle((JSONObject) arguments);
+              platformMessageHandler.setSystemUiOverlayStyle(systemChromeStyle);
+              result.success(null);
+            } catch (JSONException | NoSuchFieldException exception) {
+              // JSONException: One or more expected fields were either omitted or referenced an invalid type.
+              // NoSuchFieldException: One or more of the brightness names are invalid.
+              result.error("error", exception.getMessage(), null);
+            }
+            break;
+          case "SystemNavigator.pop":
+            platformMessageHandler.popSystemNavigator();
+            result.success(null);
+            break;
+          case "SystemGestures.setSystemGestureExclusionRects":
+            if (!(arguments instanceof JSONArray)) {
+              String inputTypeError = "Input type is incorrect. Ensure that a List<Map<String, int>> is passed as the input for SystemGestureExclusionRects.setSystemGestureExclusionRects.";
+              result.error("inputTypeError", inputTypeError, null);
+              break;
+            }
+
+            Log.v("Got here", arguments.toString());
```

Comment:
This log probably isn't useful and should be removed, even at the `verbose` level.

Reply:
This was an artifact from when I was testing my tests. I'll remove it!